.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["env"],
```

Comment:
I thought `babel-preset-env` required a `.browserslistrc` to work.  Does it have some sort of default?